tools/cbmc/proofs/HTTP/IotHttpsClient_AddHeader/IotHttpsClient_AddHeader_harness.c

```diff
@@ -45,15 +45,17 @@ void harness() {
   IotHttpsRequestHandle_t reqHandle = allocate_IotRequestHandle();
   if (reqHandle)
     __CPROVER_assume(is_valid_IotRequestHandle(reqHandle));
-  uint32_t nameLen;
-  uint32_t valueLen;
+
```

Comment:
Why the new blank line?

Reply:
I just thought it makes it cleaner and readable --- I added line breaks between:
(1) the initialization of the RequestHandle object itself, (2) the initialization of the header name and value, (3) the actual function call (and assertions, but we don't have any).